docker-storage-setup.1

```diff
@@ -36,9 +36,10 @@ STORAGE_DRIVER:
       overlay2 and "". Empty string tells docker-storage-setup to
       not do any storage setup.
 
-EXTRA_DOCKER_STORAGE_OPTIONS:
+EXTRA_STORAGE_OPTIONS:
       A set of extra options that should be passed to the Docker
```

Comment:
I think docker-storage-setup will have to know about the environment for which storage is being generated. Because output file format will depend on that. For example, --storage-opt key=value is docker format and other run time might have different format.